src/module-vsbridge-indexer-core/Cache/Processor.php

```diff
@@ -129,15 +125,22 @@ public function cleanCacheByTags($storeId, array $tags)
     {
         $storeId = (int) $storeId;
 
-        if ($this->config->clearCache($storeId)) {
-            $cacheTags = implode(',', $tags);
-            $cacheInvalidateUrl = $this->getInvalidateCacheUrl($storeId) . $cacheTags;
+        $cacheTags = array_intersect($tags, $this->getCacheTags());
```

Comment:
Shouldn't this be: 
`$cacheTags = array_intersect_key($this->getCacheTags(), array_flip($tags));`